test/e2e/networks_regressions/slow_recheck.toml

```diff
@@ -0,0 +1,217 @@
+ipv6 = false
+initial_height = 0
+disable_pex = false
+key_type = ""
+abci_protocol = "tcp"
+prepare_proposal_delay = "200ms"
+process_proposal_delay = "200ms"
+check_tx_delay = "120ms"
```

Comment:
It would be nice to start adding comments in these manifests, like say this parameters is responsible for the slow rechecks

Reply:
Good remark. I added a comment to the manifest file.